BankingKataSpecs/AccountSpecs/Features/CreateAccount.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: Create account
+	In order to us my account
+	As a foo user
+	I want to be able to create an account.
+
+@mytag
+Scenario: New account has zero cash.
+	Given I want to create an account
```

Comment:
It is redundant, you'll see in it's step that there's a TODO: Do I need to do anything here?